src/Aframe/arcball-controller.js

```diff
@@ -90,24 +90,24 @@ THREE.TrackballControls = function (object, domElement) {
 
   // events
 
-  var changeEvent = { type: "change" };
-  var startEvent = { type: "start" };
-  var endEvent = { type: "end" };
+  let changeEvent = { type: "change" };
+  let startEvent = { type: "start" };
+  let endEvent = { type: "end" };
 
   // methods
 
   this.handleResize = function () {
-    var box = this.domElement.getBoundingClientRect();
+    let box = this.domElement.getBoundingClientRect();
```

Comment:
Also `const`? Anything that isn't getting reassigned should be `const`. When I'm programming in javascript `const` is my default and I use `let` only when things need to be reassigned

Reply:
I'm gonna stop leaving `const` notes now ;-)